make-coe/make-coe.go

```diff
@@ -170,7 +191,7 @@ func (carina *MakeCOE) DeleteCluster(token string) (common.Cluster, error) {
 			}
 		}
 
-		return nil, errors.Wrap(err, fmt.Sprintf("[make-coe] Unable to delete cluster (%s)", token))
+		return nil, handleLibcarinaError(errors.Wrap(err, fmt.Sprintf("[make-coe] Unable to delete cluster (%s)", token)))
```

Comment:
Bah, GitHub won't let me comment on a line you didn't change. 😄 On line 185, I am wondering if we should be using errors.Cause()? It won't hurt, and is safer now that we are wrapping errors in more places.